Dockerfile_x64

```diff
@@ -0,0 +1,39 @@
+FROM ultralytics/ultralytics:latest
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV TZ="Europe/Amsterdam"
+
+RUN apt-get update  \
+    && apt-get upgrade -y --fix-missing \
+    && apt-get install -y --no-install-recommends \
+    tzdata \
+    python3-pip \
+    python3-venv \
+    nano \
+    python3-dev \
+    build-essential \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+ENV PATH=/root/.local/bin:$PATH
+
+WORKDIR /usr/src
+
+COPY requirements_on_edge.txt .
+
+# Need to install CVToolkit separately to prevent it from re-installing Torch
+RUN uv pip install --system "setuptools<71.0.0"
```

Comment:
Does uv come pre-installed in the ultralytics image? If so, maybe we can also use it for the device docker :)